src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+$grey: rgba(97, 96, 112, 1);
+$main_accent: rgba(6, 11, 53, 1);
+$blue: rgba(0, 172, 220, 1);
+$hover_color: rgba(243, 243, 243, 1);
```

Comment:
i think will be good to move variables into separated file, like `variables.css`